src/test/scala/nl/vroste/zio/kinesis/client/DynamicConsumerTest.scala

```diff
@@ -215,35 +220,41 @@ object DynamicConsumerTest extends DefaultRunnableSpec {
         } yield stream)
 
       createStream(streamName, 2)
-        .provideSomeLayer[Console](env)
+        .provideSomeLayer[Console](env.fresh)
```

Comment:
Does the `fresh` help here?

Reply:
no I tried it and it does not unfortunately.